docs/learn/learn-nominator.md

```diff
@@ -49,18 +49,91 @@ nomination. However, the election algorithm attempts to minimize this situation,
 occur often, so you should almost always see only a single active nomination per era. See the
 [section on Phragmén optimization](learn-phragmen.md#optimizations) for more details.
 
+### Bags-list
+
+Nominating accounts are placed in a semi-sorted list called bags-list. This sorting functionality is
+extremely important for the
+[long-term improvements](https://gist.github.com/kianenigma/aa835946455b9a3f167821b9d05ba376) of the
+staking/election system. Bags-list allows up to 50,000 nominators to set their _intention_ to
+nominate, of which, the stake of the top 22,500 nominators is considered for elections that
```

Comment:
should we use the term "electing" and reference to where it is explained?

Reply:
Will do